### PR TITLE
Add sadd_returns_boolean + sadd? method

### LINKED ISSUE #216: Deprecation warning: `Redis#sadd` will always return an Integer in Redis 5.0.0. Use `Redis#sadd?` instead
I am using **redis 4.8.0** and **redis-namespace 1.9.0** and noticed this deprecation warning.

### PR DESCRIPTION
**Pull request by radar**
Fixes #216.

### CODE REVIEW

File: lib/redis/namespace.rb [hunk 1]
```diff
@@ -239,6 +240,12 @@ class Namespace
     # Support 1.8.7 by providing a namespaced reference to Enumerable::Enumerator
     Enumerator = Enumerable::Enumerator unless defined?(::Enumerator)
 
+    @sadd_returns_boolean = true
+
+    class << self
+      attr_accessor :sadd_returns_boolean
+    end
+
```
Comment (iloveitaly) [inline lib/redis/namespace.rb:248]: This isn't needed, right? Can we remove it?

**Review by iloveitaly — CHANGES_REQUESTED** [review]

File: lib/redis/namespace.rb (hunk 1 above)
Comment (radar) [reply lib/redis/namespace.rb:248]: It's needed, as it provides the way to set and read the `sadd_returns_boolean` value. This is used by the `Redis` gem to determine whether or not to display that deprecation message.

**Comment by radar** [comment]
@iloveitaly Is there anything else you'd like me to look into for this PR?

File: lib/redis/namespace.rb [hunk 2]
```diff
@@ -239,6 +240,12 @@ class Namespace
     # Support 1.8.7 by providing a namespaced reference to Enumerable::Enumerator
     Enumerator = Enumerable::Enumerator unless defined?(::Enumerator)
 
+    @sadd_returns_boolean = true
```
Comment (iloveitaly) [inline lib/redis/namespace.rb:243]:
```suggestion
    # This is used by the Redis gem to determine whether or not to display that deprecation message.
    @sadd_returns_boolean = true
```

**Comment by radar** [comment]
Thanks for the merge! Can we please get this included in a new gem release?

**Comment by iloveitaly** [comment]
Yes! I'll try to get to it this week. Ping me if it's not done!

**Comment by PatrickTulskie** [comment]
@iloveitaly if you're busy I should be able to knock out a release today/tomorrow.

**Comment by iloveitaly** [comment]
Yeah I'm totally slammed so if you have time that would be great

**Comment by PatrickTulskie** [comment]
Release is mostly done but rubygems is kicking me out. I'll push up the gem once @iloveitaly has a chance to take a look.

**Comment by iloveitaly** [comment]
I think I fixed it! Let me know if that doesn't work.

**Comment by PatrickTulskie** [comment]
Thanks @iloveitaly

@radar this is now in 1.10.0. Thanks for sorting this out!

**Pull request merged**